js/zb.js

```diff
@@ -156,74 +168,138 @@ module.exports = class zb extends Exchange {
                 },
             },
             'api': {
-                'trade': {
-                    'get': [
-                        'getFeeInfo', // withdrawal fees
-                    ],
-                },
-                'public': {
-                    'get': [
-                        'markets',
-                        'ticker',
-                        'allTicker',
-                        'depth',
-                        'trades',
-                        'kline',
-                        'getGroupMarkets',
-                    ],
+                'spot': {
+                    'v1': {
+                        'public': {
+                            'get': [
+                                'markets',
+                                'ticker',
+                                'allTicker',
+                                'depth',
+                                'trades',
+                                'kline',
+                                'getGroupMarkets',
+                            ],
+                        },
+                        'private': {
+                            'get': [
+                                // spot API
+                                'order',
+                                'orderMoreV2',
+                                'cancelOrder',
+                                'getFeeInfo',
```

Comment:
The `getFeeInfo` endpoint is actually public, se we will need to move it to the public section above.

You can address this URL in the browser: https://trade.zb.work/api/getFeeInfo

```
~/ccxt $ python3 examples/py/cli.py zb tradeGetGetFeeInfo --verbose
Python v3.8.2
CCXT v1.72.98

fetch Request: zb GET https://trade.zb.work/api/getFeeInfo RequestHeaders: {'User-Agent': 'python-requests/2.25.1', 'Accept-Encoding': 'gzip, deflate'} RequestBody: None

fetch Response: zb GET https://trade.zb.work/api/getFeeInfo 200 ResponseHeaders: {'Server': 'nginx', 'Date': 'Sun, 13 Feb 2022 23:53:26 GMT', 'Content-Type': 'application/json;charset=utf-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Vary': 'Accept-Encoding', 'CNDServer': 'cache_server_1', 'Set-Cookie': 'zJSESSIONID=2A7FC5296B3DCAEB58290E07052C4F73; Domain=zb.work; Path=/; HttpOnly', 'Content-Encoding': 'gzip'} ResponseBody: {"code":1000,"message":"success","result":{"AGLD":[{"chainName":"AGLD","canWithdraw":true,"fee":25.0,"mainChainName":"ETH","canDeposit":true}],"MATIC":[{"chainName":"MATIC","canWithdraw":true,"fee":15.0,"mainChainName":"ETH","canDeposit":true}],"CDC":[{"chainName":"CDC","canWithdraw":false,"fee":1000.0,"mainChainName":"ETH","canDeposit":false}],"LVN":[{"chainName":"LVN","canWithdraw":true,"fee":200000.0,"mainChainName":"ETH","canDeposit":true}],"SHIB":[{"chainName":"SHIB","canWithdraw":true,"fee":1400000.0,"mainChainName":"ETH","canDeposit":true}],"TVK":[{"chainName":"TVK","canWithdraw":true,"fee":160.0,"mainChainName":"ETH","canDeposit":true}],"IOTX":[{"chainName":"IOTX","canWithdraw":true,"fee":0.1,"mainChainName":"IOTX","canDeposit":true}],"GLM":[{"chainName":"GLM","canWithdraw":true,"fee":20.0,"mainChainName":"ETH","canDeposit":true}],"CERE":[{"chainName":"CERE","canWithdraw":true,"fee":600.0,"mainChainName":"ETH","canDeposit":true}],"DFL":[{"chainName":"BEP20","canWithdraw":true,"fee":10.0,"mainChainName":"BITBSC","canDeposit":true},{"chainName":"ERC20","canWithdraw":true,"fee":150.0,"mainChainName":"ETH","canDeposit":true}],"DAWN":[{"chainName":"DAWN","canWithdraw":true,"fee":20.0,"mainChainName":"ETH","canDeposit":true}],"NEAR":[{"chainName":"NEAR","canWithdraw":true,"fee":0.1,"mainChainName":"NEAR","canDeposit":true}],"HNS":[{"chainName":"HNS","canWithdraw":true,"fee":0.2,"mainChainName":"HNS","canDeposit":true}],"FOMP":[{"chainName":"FOMP","canWithdraw":true,"fee":10.0,"mainChainName":"HT","canDeposit":true}],"AUDIO":[{"chainName":"AUDIO","canWithdraw":true,"fee":25.0,"mainChainName":"ETH","canDeposit":true}],"GLS":[{"chainName":"GLS","canWithdraw":true,"fee":0.1,"mainChainName":"GLS","canDeposit":true}],"QC":[{"canWithdraw":true,"fee":5.0,"mainChainName":"TRUE","canDeposit":true},{"chainName":"ERC20","canWithdraw":true,"fee":20.0,"mainChainName":"ETH","canDeposit":true},{"chainName":"QRC20","canWithdraw":true,"fee":5.0,"mainChainName":"QTUM","canDeposit":true}],"FARM":[{"chainName":"FARM","canWithdraw":false,"fee":0.3,"mainChainName":"ETH","canDeposit":true}],"DENT":[{"chainName":"DENT","canWithdraw":true,"fee":13000.0,"mainChainName":"ETH","canDeposit":true}],"bitCNY":[{"chainName":"bitCNY","canWithdraw":true,"fee":20.0,"mainChainName":"BTS","canDeposit":true}],"SDN":[{"chainName":"SDN","canWithdraw":true,"fee":1.0,"mainChainName":"SDN","canDeposit":true}],"ZRX":[{"chainName":"ZRX","canWithdraw":true,"fee":10.0,"mainChainName":"ETH","canDeposit":true}],"SUB":[{"chainName":"SUB","canWithdraw":true,"fee":20.0,"mainChainName":"ETH","canDeposit":true}],"AAA":[{"chainName":"AAA","canWithdraw":true,"fee":1500.0,"mainChainName":"ETH","canDeposit":true}],"BCD":[{"chainName":"BCD","canWithdraw":false,"fee":0.001,"mainChainName":"BTC","canDeposit":false}],"AD":[{"chainName":"AD","canWithdraw":false,"fee":0.01,"mainChainName":"MAIN-AD","canDeposit":false}],"AE":[{"chainName":"AE","canWithdraw":false,"fee":0.1,"mainChainName":"AE","canDeposit":false}],"BSV":[{"chainName":"BSV","canWithdraw":false,"fee":0.001,"mainChainName":"BSV","canDeposit":false}],"YTNB":[{"chainName":"YTNB","canWithdraw":true,"fee":8.0,"mainChainName":"ETH","canDeposit":true}],"BCH":[{"chainName":"BCH","canWithdraw":true,"fee":5.0E-4,"mainChainName":"BCH","canDeposit":true}],"HOP":[{"chainName":"HOP","canWithdraw":true,"fee":14.0,"mainChainName":"ETH","canDeposit":true}],"OMG":[{"chainName":"OMG","canWithdraw":false,"fee":3.0,"mainChainName":"ETH","canDeposit":true}],"NKN":[{"chainName":"NKN","canWithdraw":true,"fee":120.0,"mainChainName":"ETH","canDeposit":true}],"RSR":[{"chainName":"RSR","canWithdraw":true,"fee":1500.0,"mainChainName":"ETH","canDeposit":true}],"BTC":[{"chainName":"BTC","canWithdraw":true,"fee":5.0E-4,"mainChainName":"BTC","canDeposit":true}],"MIR":[{"chainName":"MIR","canWithdraw":true,"fee":15.0,"mainChainName":"ETH","canDeposit":true}],"BCW":[{"chainName":"BCW","canWithdraw":true,"fee":30.0,"mainChainName":"ETH","canDeposit":false}],"BTH":[{"chainName":"BTH","canWithdraw":false,"fee":0.1,"mainChainName":"BTH","canDeposit":false}],"BTM":[{"chainName":"BTM","canWithdraw":true,"fee":10.0,"mainChainName":"BTM","canDeposit":true}],"RCN":[{"chainName":"RCN","canWithdraw":true,"fee":6000.0,"mainChainName":"ETH","canDeposit":true}],"BTN":[{"chainName":"BTN","canWithdraw":false,"fee":0.005,"mainChainName":"BTC","canDeposit":true}],"BTP":[{"chainName":"BTP","canWithdraw":false,"fee":0.001,"mainChainName":"BTP","canDeposit":false}],"WMT":[{"chainName":"WMT","canWithdraw":false,"fee":1.0,"mainChainName":"ADA","canDeposit":false}],"BTS":[{"chainName":"BTS","canWithdraw":true,"fee":3.0,"mainChainName":"BTS","canDeposit":true}],"CVX":[{"chainName":"CVX","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"RTF":[{"chainName":"RTF","canWithdraw":true,"fee":0.05,"mainChainName":"TRUE","canDeposit":true}],"LBTC":[{"chainName":"LBTC","canWithdraw":false,"fee":1.0E-4,"mainChainName":"LBTC","canDeposit":true}],"GNO":[{"chainName":"GNO","canWithdraw":false,"fee":0.01,"mainChainName":"ETH","canDeposit":true}],"ANKR":[{"chainName":"ANKR","canWithdraw":true,"fee":400.0,"mainChainName":"ETH","canDeposit":true}],"BDM":[{"chainName":"BDM","canWithdraw":true,"fee":30000.0,"mainChainName":"ETH","canDeposit":true}],"SUSHI":[{"chainName":"SUSHI","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":true}],"M":[{"chainName":"M","canWithdraw":false,"fee":0.01,"mainChainName":"M","canDeposit":false}],"ALGO":[{"chainName":"ALGO","canWithdraw":true,"fee":0.2,"mainChainName":"ALGO","canDeposit":true}],"BDS":[{"chainName":"BDS","canWithdraw":true,"fee":100.0,"mainChainName":"EOS","canDeposit":true}],"HPY":[{"chainName":"HPY","canWithdraw":true,"fee":100.0,"mainChainName":"QTUM","canDeposit":true}],"THG":[{"chainName":"THG","canWithdraw":true,"fee":0.4,"mainChainName":"BSC","canDeposit":true}],"CFX":[{"chainName":"CFX","canWithdraw":true,"fee":0.5,"mainChainName":"CFX","canDeposit":true}],"ONT":[{"chainName":"ONT","canWithdraw":true,"fee":1.0,"mainChainName":"MAIN-ONT","canDeposit":true}],"TYB":[{"chainName":"TYB","canWithdraw":true,"fee":1.0,"mainChainName":"TRX","canDeposit":true}],"DIA":[{"chainName":"DIA","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"BED":[{"chainName":"BED","canWithdraw":true,"fee":0.5,"mainChainName":"BED","canDeposit":true}],"ACC":[{"chainName":"ACC","canWithdraw":false,"fee":0.1,"mainChainName":"MAIN-ACC","canDeposit":false}],"OOE":[{"chainName":"OOE","canWithdraw":true,"fee":10.0,"mainChainName":"BSC","canDeposit":true}],"MANA":[{"chainName":"MANA","canWithdraw":true,"fee":15.0,"mainChainName":"ETH","canDeposit":true}],"GUSDT":[{"chainName":"GUSDT","canWithdraw":true,"fee":2000.0,"mainChainName":"ETH","canDeposit":false}],"ACH":[{"chainName":"ACH","canWithdraw":true,"fee":600.0,"mainChainName":"ETH","canDeposit":true}],"DIP":[{"chainName":"DIP","canWithdraw":true,"fee":1000.0,"mainChainName":"ETH","canDeposit":false}],"CSPR":[{"chainName":"CSPR","canWithdraw":true,"fee":1.0,"mainChainName":"CSPR","canDeposit":true}],"SGB":[{"chainName":"SGB","canWithdraw":true,"fee":10.0,"mainChainName":"SGB","canDeposit":true}],"STAR":[{"chainName":"STAR","canWithdraw":false,"fee":0.1,"mainChainName":"STAR","canDeposit":false}],"PAF":[{"chainName":"PAF","canWithdraw":true,"fee":1600.0,"mainChainName":"ETH","canDeposit":true}],"1000HOKK":[{"chainName":"1000HOKK","canWithdraw":true,"fee":2.0E7,"mainChainName":"ETH","canDeposit":false}],"GRAM":[{"chainName":"GRAM","canWithdraw":true,"fee":10000.0,"mainChainName":"ETH","canDeposit":true}],"ICP":[{"chainName":"ICP","canWithdraw":true,"fee":0.01,"mainChainName":"ICP","canDeposit":true}],"REN":[{"chainName":"REN","canWithdraw":false,"fee":60.0,"mainChainName":"ETH","canDeposit":true}],"ADA":[{"chainName":"ADA","canWithdraw":true,"fee":1.0,"mainChainName":"ADA","canDeposit":true}],"REQ":[{"chainName":"REQ","canWithdraw":true,"fee":77.0,"mainChainName":"ETH","canDeposit":true}],"ICX":[{"chainName":"ICX","canWithdraw":true,"fee":3.0,"mainChainName":"ICX","canDeposit":true}],"STORJ":[{"chainName":"STORJ","canWithdraw":true,"fee":12.0,"mainChainName":"ETH","canDeposit":true}],"RARI":[{"chainName":"RARI","canWithdraw":true,"fee":3.0,"mainChainName":"ETH","canDeposit":true}],"REV":[{"chainName":"REV","canWithdraw":true,"fee":5.0,"mainChainName":"REV","canDeposit":true}],"TV":[{"chainName":"TV","canWithdraw":false,"fee":1.0,"mainChainName":"TV","canDeposit":false}],"RARE":[{"chainName":"RARE","canWithdraw":true,"fee":40.0,"mainChainName":"ETH","canDeposit":true}],"PAX":[{"chainName":"PAX","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":true}],"DOGE":[{"chainName":"DOGE","canWithdraw":true,"fee":20.0,"mainChainName":"DOGE","canDeposit":true}],"MLN":[{"chainName":"MLN","canWithdraw":true,"fee":0.2,"mainChainName":"ETH","canDeposit":true}],"WAVES":[{"chainName":"WAVES","canWithdraw":false,"fee":0.01,"mainChainName":"WAVES","canDeposit":false}],"CHZ":[{"chainName":"CHZ","canWithdraw":true,"fee":120.0,"mainChainName":"ETH","canDeposit":true}],"XRP":[{"chainName":"XRP","canWithdraw":true,"fee":0.1,"mainChainName":"XRP","canDeposit":true}],"JASMY":[{"chainName":"JASMY","canWithdraw":true,"fee":260.0,"mainChainName":"ETH","canDeposit":true}],"IDV":[{"chainName":"IDV","canWithdraw":true,"fee":3000.0,"mainChainName":"ETH","canDeposit":true}],"LUCKY":[{"chainName":"LUCKY","canWithdraw":true,"fee":400000.0,"mainChainName":"ETH","canDeposit":true}],"ULU":[{"chainName":"ULU","canWithdraw":true,"fee":15.0,"mainChainName":"ETH","canDeposit":true}],"KAVA":[{"chainName":"KAVA","canWithdraw":true,"fee":0.1,"mainChainName":"KAVA","canDeposit":true}],"SAND":[{"chainName":"SAND","canWithdraw":true,"fee":8.0,"mainChainName":"ETH","canDeposit":true}],"BGH":[{"chainName":"BGH","canWithdraw":true,"fee":0.1,"mainChainName":"BGH","canDeposit":true}],"UZ":[{"chainName":"UZ","canWithdraw":true,"fee":0.1,"mainChainName":"TRX","canDeposit":true}],"SANA":[{"chainName":"SANA","canWithdraw":true,"fee":15000.0,"mainChainName":"ETH","canDeposit":true}],"UMA":[{"chainName":"UMA","canWithdraw":false,"fee":4.0,"mainChainName":"ETH","canDeposit":true}],"EP":[{"chainName":"EP","canWithdraw":true,"fee":2.0,"mainChainName":"EP","canDeposit":true}],"QUN":[{"chainName":"QUN","canWithdraw":true,"fee":23000.0,"mainChainName":"ETH","canDeposit":true}],"METAS":[{"chainName":"METAS","canWithdraw":true,"fee":10.0,"mainChainName":"BSC","canDeposit":true}],"TUSD":[{"chainName":"TUSD","canWithdraw":true,"fee":40.0,"mainChainName":"ETH","canDeposit":false}],"HCT":[{"chainName":"HCT","canWithdraw":true,"fee":1.0,"mainChainName":"AVAX","canDeposit":true}],"XCH":[{"chainName":"XCH","canWithdraw":true,"fee":0.0013,"mainChainName":"XCH","canDeposit":true}],"RGT":[{"chainName":"RGT","canWithdraw":true,"fee":2.3,"mainChainName":"ETH","canDeposit":true}],"VSYS":[{"chainName":"VSYS","canWithdraw":true,"fee":1.0,"mainChainName":"VSYS","canDeposit":true}],"THETA":[{"chainName":"THETA","canWithdraw":true,"fee":0.1,"mainChainName":"THETA","canDeposit":true}],"ENJ":[{"chainName":"ENJ","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":true}],"MNC":[{"chainName":"MNC","canWithdraw":true,"fee":100000.0,"mainChainName":"ETH","canDeposit":true}],"YFII":[{"chainName":"YFII","canWithdraw":true,"fee":1.0E-4,"mainChainName":"ETH","canDeposit":true}],"FN":[{"chainName":"FN","canWithdraw":false,"fee":0.0,"mainChainName":"FN","canDeposit":false}],"GRT":[{"chainName":"GRT","canWithdraw":true,"fee":60.0,"mainChainName":"ETH","canDeposit":true}],"ORN":[{"chainName":"ORN","canWithdraw":true,"fee":9.0,"mainChainName":"ETH","canDeposit":true}],"ENS":[{"chainName":"ENS","canWithdraw":true,"fee":1.0,"mainChainName":"ETH","canDeposit":true}],"NIFTSY":[{"chainName":"NIFTSY","canWithdraw":true,"fee":1500.0,"mainChainName":"ETH","canDeposit":true}],"AAVE":[{"chainName":"AAVE","canWithdraw":true,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"UNI":[{"chainName":"UNI","canWithdraw":true,"fee":1.0,"mainChainName":"ETH","canDeposit":true}],"FX":[{"chainName":"FX","canWithdraw":true,"fee":28.0,"mainChainName":"ETH","canDeposit":true}],"TLM":[{"chainName":"TLM","canWithdraw":true,"fee":200.0,"mainChainName":"ETH","canDeposit":true}],"MNU":[{"chainName":"MNU","canWithdraw":true,"fee":30.0,"mainChainName":"BSC","canDeposit":true}],"PERP":[{"chainName":"PERP","canWithdraw":true,"fee":2.1,"mainChainName":"ETH","canDeposit":true}],"B91":[{"chainName":"B91","canWithdraw":true,"fee":4.0,"mainChainName":"ETH","canDeposit":false}],"DMD":[{"chainName":"DMD","canWithdraw":true,"fee":0.03,"mainChainName":"EOS","canDeposit":true}],"BOND":[{"chainName":"BOND","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"YFI":[{"chainName":"YFI","canWithdraw":true,"fee":5.0E-4,"mainChainName":"ETH","canDeposit":true}],"XFLR":[{"chainName":"XFLR","canWithdraw":false,"fee":10.0,"mainChainName":"XFLR","canDeposit":false}],"LUNA":[{"chainName":"LUNA","canWithdraw":true,"fee":0.01,"mainChainName":"LUNA","canDeposit":true}],"XTZ":[{"chainName":"XTZ","canWithdraw":false,"fee":0.1,"mainChainName":"XTZ","canDeposit":false}],"GF":[{"chainName":"GF","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"AURORA":[{"chainName":"AURORA","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":true}],"BCHA":[{"chainName":"BCHA","canWithdraw":false,"fee":0.01,"mainChainName":"BCHA","canDeposit":false}],"PDX":[{"chainName":"PDX","canWithdraw":true,"fee":1300.0,"mainChainName":"ETH","canDeposit":true}],"XUC":[{"chainName":"XUC","canWithdraw":true,"fee":1.0,"mainChainName":"ETH","canDeposit":true}],"EOS":[{"chainName":"EOS","canWithdraw":true,"fee":0.1,"mainChainName":"EOS","canDeposit":true}],"GST":[{"chainName":"GST","canWithdraw":false,"fee":0.01,"mainChainName":"MAIN-GST","canDeposit":false}],"LQTY":[{"chainName":"LQTY","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"SWFTC":[{"chainName":"SWFTC","canWithdraw":true,"fee":30000.0,"mainChainName":"ETH","canDeposit":true}],"RACA":[{"chainName":"RACA","canWithdraw":true,"fee":1000.0,"mainChainName":"BSC","canDeposit":true}],"BICO":[{"chainName":"BICO","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"DNA":[{"chainName":"DNA","canWithdraw":true,"fee":20.0,"mainChainName":"MAIN-DNA","canDeposit":true}],"XEC":[{"chainName":"XEC","canWithdraw":true,"fee":5000.0,"mainChainName":"XEC","canDeposit":true}],"GTC":[{"chainName":"GTC","canWithdraw":true,"fee":1.7,"mainChainName":"ETH","canDeposit":true}],"YGG":[{"chainName":"YGG","canWithdraw":true,"fee":6.0,"mainChainName":"ETH","canDeposit":true}],"ENTC":[{"chainName":"ENTC","canWithdraw":false,"fee":10.0,"mainChainName":"HC","canDeposit":false}],"EPC":[{"chainName":"EPC","canWithdraw":true,"fee":40.0,"mainChainName":"QTUM","canDeposit":true}],"PEOPLE":[{"chainName":"PEOPLE","canWithdraw":true,"fee":290.0,"mainChainName":"ETH","canDeposit":true}],"HC":[{"chainName":"HC","canWithdraw":false,"fee":0.01,"mainChainName":"HC","canDeposit":false}],"AXS":[{"chainName":"AXS","canWithdraw":false,"fee":0.3,"mainChainName":"ETH","canDeposit":true}],"XEM":[{"chainName":"XEM","canWithdraw":false,"fee":4.0,"mainChainName":"XEM","canDeposit":false}],"HI":[{"chainName":"HI","canWithdraw":true,"fee":2.0,"mainChainName":"BSC","canDeposit":true}],"BZZ":[{"chainName":"BZZ","canWithdraw":true,"fee":20.0,"mainChainName":"ETH","canDeposit":true}],"TRAC":[{"chainName":"TRAC","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":false}],"WTF":[{"chainName":"WTF","canWithdraw":false,"fee":1000.0,"mainChainName":"ETH","canDeposit":true}],"DNT":[{"chainName":"DNT","canWithdraw":true,"fee":300.0,"mainChainName":"ETH","canDeposit":true}],"CLV":[{"chainName":"CLV","canWithdraw":true,"fee":50.0,"mainChainName":"ETH","canDeposit":true}],"FLOW":[{"chainName":"FLOW","canWithdraw":false,"fee":0.1,"mainChainName":"FLOW","canDeposit":false}],"COMP":[{"chainName":"COMP","canWithdraw":true,"fee":0.05,"mainChainName":"ETH","canDeposit":true}],"HX":[{"chainName":"HX","canWithdraw":true,"fee":0.1,"mainChainName":"HX","canDeposit":true}],"CATE-OLD":[{"chainName":"CATE-OLD","canWithdraw":true,"fee":2000000.0,"mainChainName":"BSC","canDeposit":false}],"BGPT":[{"chainName":"BGPT","canWithdraw":true,"fee":200000.0,"mainChainName":"ETH","canDeposit":true}],"SLP":[{"chainName":"SLP","canWithdraw":true,"fee":1500.0,"mainChainName":"ETH","canDeposit":true}],"SLT":[{"chainName":"SLT","canWithdraw":true,"fee":100.0,"mainChainName":"ETH","canDeposit":true}],"CMD":[{"chainName":"CMD","canWithdraw":true,"fee":0.1,"mainChainName":"CMD","canDeposit":true}],"NBS":[{"chainName":"NBS","canWithdraw":true,"fee":5.0,"mainChainName":"NBS","canDeposit":true}],"BKH":[{"chainName":"BKH","canWithdraw":true,"fee":1.0,"mainChainName":"TRX","canDeposit":true}],"FORTH":[{"chainName":"FORTH","canWithdraw":true,"fee":4.0,"mainChainName":"ETH","canDeposit":true}],"DOM":[{"chainName":"DOM","canWithdraw":true,"fee":0.1,"mainChainName":"DOM","canDeposit":true}],"XWC":[{"chainName":"XWC","canWithdraw":true,"fee":0.01,"mainChainName":"MAIN-XWC","canDeposit":true}],"ZB":[{"canWithdraw":true,"fee":5.0,"mainChainName":"TRUE","canDeposit":true},{"chainName":"ERC20","canWithdraw":true,"fee":10.0,"mainChainName":"ETH","canDeposit":true}],"CENNZ":[{"chainName":"CENNZ","canWithdraw":false,"fee":5.0,"mainChainName":"MAIN-CENNZ","canDeposit":false}],"DOT":[{"chainName":"DOT","canWithdraw":true,"fee":0.1,"mainChainName":"DOT","canDeposit":true}],"VBT":[{"chainName":"VBT","canWithdraw":true,"fee":400000.0,"mainChainName":"ETH","canDeposit":true}],"1000KISHU":[{"chainName":"1000KISHU","canWithdraw":true,"fee":4000000.0,"mainChainName":"ETH","canDeposit":true}],"1INCH":[{"chainName":"1INCH","canWithdraw":true,"fee":15.0,"mainChainName":"ETH","canDeposit":true}],"AVAX":[{"chainName":"AVAX","canWithdraw":true,"fee":0.1,"mainChainName":"AVAX","canDeposit":true}],"MITH":[{"chainName":"MITH","canWithdraw":true,"fee":500.0,"mainChainName":"ETH","canDeposit":true}],"ZKN":[{"chainName":"ZKN","canWithdraw":true,"fee":100.0,"mainChainName":"BSC","canDeposit":true}],"NCT":[{"chainName":"NCT","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"POWR":[{"chainName":"POWR","canWithdraw":true,"fee":90.0,"mainChainName":"ETH","canDeposit":true}],"CART":[{"chainName":"CART","canWithdraw":true,"fee":180.0,"mainChainName":"ETH","canDeposit":true}],"KNC":[{"chainName":"KNC","canWithdraw":true,"fee":10.0,"mainChainName":"ETH","canDeposit":true}],"KILT":[{"chainName":"KILT","canWithdraw":true,"fee":0.1,"mainChainName":"KILT","canDeposit":true}],"CHAT":[{"chainName":"CHAT","canWithdraw":true,"fee":30000.0,"mainChainName":"ETH","canDeposit":true}],"RLC":[{"chainName":"RLC","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"ATOM":[{"chainName":"ATOM","canWithdraw":true,"fee":0.01,"mainChainName":"ATOM","canDeposit":true}],"QUICK":[{"chainName":"QUICK","canWithdraw":false,"fee":0.15,"mainChainName":"ETH","canDeposit":true}],"CATE":[{"chainName":"CATE","canWithdraw":true,"fee":3000000.0,"mainChainName":"BSC","canDeposit":true}],"LPT":[{"chainName":"LPT","canWithdraw":true,"fee":1.0,"mainChainName":"ETH","canDeposit":true}],"WNCG":[{"chainName":"WNCG","canWithdraw":true,"fee":20.0,"mainChainName":"ETH","canDeposit":true}],"SNT":[{"chainName":"SNT","canWithdraw":true,"fee":60.0,"mainChainName":"ETH","canDeposit":true}],"SNX":[{"chainName":"SNX","canWithdraw":true,"fee":0.5,"mainChainName":"ETH","canDeposit":true}],"RLY":[{"chainName":"RLY","canWithdraw":true,"fee":100.0,"mainChainName":"ETH","canDeposit":true}],"FUN":[{"chainName":"FUN","canWithdraw":true,"fee":3000.0,"mainChainName":"ETH","canDeposit":true}],"BTRST":[{"chainName":"BTRST","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":false}],"ROSE":[{"chainName":"ROSE","canWithdraw":true,"fee":0.001,"mainChainName":"ROSE","canDeposit":true}],"WAXP":[{"chainName":"WAXP","canWithdraw":true,"fee":5.0,"mainChainName":"WAXP","canDeposit":true}],"GLMR":[{"chainName":"GLMR","canWithdraw":false,"fee":0.1,"mainChainName":"GLMR","canDeposit":true}],"SUKU":[{"chainName":"SUKU","canWithdraw":true,"fee":60.0,"mainChainName":"ETH","canDeposit":true}],"XYM":[{"chainName":"XYM","canWithdraw":true,"fee":5.0,"mainChainName":"XYM","canDeposit":true}],"DAI":[{"chainName":"DAI","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"XYO":[{"chainName":"XYO","canWithdraw":true,"fee":1000.0,"mainChainName":"ETH","canDeposit":true}],"GRIN":[{"chainName":"GRIN","canWithdraw":true,"fee":1.0,"mainChainName":"GRIN","canDeposit":true}],"SOL":[{"chainName":"SOL","canWithdraw":true,"fee":0.01,"mainChainName":"SOL","canDeposit":true}],"BNA":[{"chainName":"BNA","canWithdraw":true,"fee":20000.0,"mainChainName":"ETH","canDeposit":true}],"DAR":[{"chainName":"DAR","canWithdraw":true,"fee":23.0,"mainChainName":"BSC","canDeposit":true}],"FET":[{"chainName":"FET","canWithdraw":true,"fee":80.0,"mainChainName":"ETH","canDeposit":true}],"ETC":[{"chainName":"ETC","canWithdraw":false,"fee":0.01,"mainChainName":"ETC","canDeposit":false}],"SOS":[{"chainName":"SOS","canWithdraw":true,"fee":6000000.0,"mainChainName":"ETH","canDeposit":true}],"OGN":[{"chainName":"OGN","canWithdraw":true,"fee":60.0,"mainChainName":"ETH","canDeposit":true}],"BTMC":[{"chainName":"BTMC","canWithdraw":false,"fee":10.0,"mainChainName":"BTMC","canDeposit":false}],"CELR":[{"chainName":"CELR","canWithdraw":true,"fee":500.0,"mainChainName":"ETH","canDeposit":true}],"MCO":[{"chainName":"MCO","canWithdraw":true,"fee":3.0,"mainChainName":"ETH","canDeposit":false}],"ETH":[{"chainName":"ETH","canWithdraw":true,"fee":0.01,"mainChainName":"ETH","canDeposit":true}],"NEO":[{"chainName":"NEO","canWithdraw":false,"fee":0.0,"mainChainName":"NEO","canDeposit":false}],"CELO":[{"chainName":"CELO","canWithdraw":true,"fee":0.001,"mainChainName":"CELO","canDeposit":true}],"AUCTION":[{"chainName":"AUCTION","canWithdraw":true,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"LAT":[{"chainName":"LAT","canWithdraw":false,"fee":1.0,"mainChainName":"LAT","canDeposit":true}],"KPG":[{"chainName":"KPG","canWithdraw":false,"fee":0.05,"mainChainName":"KPG","canDeposit":false}],"MTL":[{"chainName":"MTL","canWithdraw":false,"fee":3.0,"mainChainName":"ETH","canDeposit":true}],"TOPC":[{"chainName":"TOPC","canWithdraw":true,"fee":30000.0,"mainChainName":"ETH","canDeposit":true}],"USDT":[{"chainName":"TRC20","canWithdraw":true,"fee":1.0,"mainChainName":"TRX","canDeposit":true},{"chainName":"OMNI","canWithdraw":true,"fee":5.0,"mainChainName":"BTC","canDeposit":true},{"chainName":"ERC20","canWithdraw":true,"fee":25.0,"mainChainName":"ETH","canDeposit":true}],"BNT":[{"chainName":"BNT","canWithdraw":true,"fee":6.0,"mainChainName":"ETH","canDeposit":true}],"ETZ":[{"chainName":"ETZ","canWithdraw":true,"fee":0.01,"mainChainName":"ETZ","canDeposit":true}],"DASH":[{"chainName":"DASH","canWithdraw":true,"fee":0.002,"mainChainName":"DASH","canDeposit":true}],"BNX":[{"chainName":"BNX","canWithdraw":false,"fee":0.04,"mainChainName":"BSC","canDeposit":true}],"SWRV":[{"chainName":"SWRV","canWithdraw":true,"fee":200.0,"mainChainName":"ETH","canDeposit":true}],"EDO":[{"chainName":"EDO","canWithdraw":false,"fee":4.0,"mainChainName":"ETH","canDeposit":false}],"ILV":[{"chainName":"ILV","canWithdraw":false,"fee":0.05,"mainChainName":"ETH","canDeposit":true}],"DSF":[{"chainName":"DSF","canWithdraw":true,"fee":800000.0,"mainChainName":"ETH","canDeposit":true}],"TRU":[{"chainName":"TRU","canWithdraw":true,"fee":130.0,"mainChainName":"ETH","canDeposit":true}],"AMC":[{"chainName":"ERC20","canWithdraw":true,"fee":200.0,"mainChainName":"ETH","canDeposit":true}],"HOTC":[{"chainName":"HOTC","canWithdraw":true,"fee":1500000.0,"mainChainName":"ETH","canDeposit":true}],"TRX":[{"chainName":"TRX","canWithdraw":true,"fee":1.0,"mainChainName":"TRX","canDeposit":true}],"UBTC":[{"chainName":"UBTC","canWithdraw":false,"fee":0.001,"mainChainName":"UBTC","canDeposit":false}],"GUSD":[{"chainName":"GUSD","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":false}],"AMP":[{"chainName":"AMP","canWithdraw":true,"fee":600.0,"mainChainName":"ETH","canDeposit":true}],"KEEP":[{"chainName":"KEEP","canWithdraw":true,"fee":45.0,"mainChainName":"ETH","canDeposit":true}],"TRUE":[{"chainName":"TRUE","canWithdraw":true,"fee":1.0,"mainChainName":"TRUE","canDeposit":true}],"NWT":[{"chainName":"NWT","canWithdraw":true,"fee":20.0,"mainChainName":"TRX","canDeposit":true}],"SAFE":[{"chainName":"SAFE","canWithdraw":true,"fee":1.0,"mainChainName":"SAFE","canDeposit":true}],"QETH":[{"chainName":"QETH","canWithdraw":false,"fee":0.01,"mainChainName":"ETH","canDeposit":false}],"JUSTICE":[{"chainName":"JUSTICE","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"TSR":[{"chainName":"TSR","canWithdraw":true,"fee":5000.0,"mainChainName":"ETH","canDeposit":true}],"LIKE":[{"chainName":"LIKE","canWithdraw":true,"fee":1.0,"mainChainName":"LIKE","canDeposit":true}],"IMX":[{"chainName":"IMX","canWithdraw":true,"fee":15.0,"mainChainName":"ETH","canDeposit":true}],"XWCC":[{"chainName":"XWCC","canWithdraw":false,"fee":1.0,"mainChainName":"XWCC","canDeposit":false}],"CRF":[{"chainName":"CRF","canWithdraw":true,"fee":400.0,"mainChainName":"ETH","canDeposit":true}],"KAN":[{"chainName":"KAN","canWithdraw":true,"fee":350.0,"mainChainName":"ETH","canDeposit":true}],"LTC":[{"chainName":"LTC","canWithdraw":true,"fee":0.005,"mainChainName":"LTC","canDeposit":true}],"USDC":[{"chainName":"ERC20","canWithdraw":true,"fee":30.0,"mainChainName":"ETH","canDeposit":true},{"chainName":"TRC20","canWithdraw":true,"fee":1.0,"mainChainName":"TRX","canDeposit":true}],"AKITA":[{"chainName":"AKITA","canWithdraw":true,"fee":3.0E7,"mainChainName":"ETH","canDeposit":true}],"HLC":[{"chainName":"HLC","canWithdraw":true,"fee":100.0,"mainChainName":"ETH","canDeposit":true}],"LTG":[{"chainName":"LTG","canWithdraw":false,"fee":0.6,"mainChainName":"LTG","canDeposit":false}],"LCX":[{"chainName":"LCX","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":false}],"LOOKS":[{"chainName":"LOOKS","canWithdraw":true,"fee":7.0,"mainChainName":"ETH","canDeposit":true}],"CRO":[{"chainName":"CRO","canWithdraw":true,"fee":0.01,"mainChainName":"CRO","canDeposit":true}],"O3":[{"chainName":"O3","canWithdraw":true,"fee":50.0,"mainChainName":"ETH","canDeposit":true}],"PLA":[{"chainName":"PLA","canWithdraw":true,"fee":20.0,"mainChainName":"ETH","canDeposit":true}],"KRL":[{"chainName":"KRL","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":false}],"CRV":[{"chainName":"CRV","canWithdraw":true,"fee":10.0,"mainChainName":"ETH","canDeposit":true}],"VGX":[{"chainName":"VGX","canWithdraw":false,"fee":13.0,"mainChainName":"ETH","canDeposit":true}],"NU":[{"chainName":"NU","canWithdraw":false,"fee":75.0,"mainChainName":"ETH","canDeposit":true}],"INK":[{"chainName":"INK","canWithdraw":true,"fee":60.0,"mainChainName":"QTUM","canDeposit":true}],"CRU":[{"chainName":"CRU","canWithdraw":true,"fee":4.0,"mainChainName":"ETH","canDeposit":true}],"TRIBE":[{"chainName":"TRIBE","canWithdraw":true,"fee":29.0,"mainChainName":"ETH","canDeposit":true}],"EFI":[{"chainName":"EFI","canWithdraw":true,"fee":50.0,"mainChainName":"ETH","canDeposit":true}],"DYDX":[{"chainName":"DYDX","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":true}],"DDM":[{"chainName":"DDM","canWithdraw":true,"fee":0.2,"mainChainName":"DDM","canDeposit":true}],"1BOX":[{"chainName":"1BOX","canWithdraw":true,"fee":100.0,"mainChainName":"BSC","canDeposit":true}],"SDOG":[{"chainName":"SDOG","canWithdraw":true,"fee":1.0E8,"mainChainName":"ETH","canDeposit":false}],"SRM":[{"chainName":"SRM","canWithdraw":true,"fee":10.0,"mainChainName":"ETH","canDeposit":true}],"POND":[{"chainName":"POND","canWithdraw":true,"fee":500.0,"mainChainName":"ETH","canDeposit":true}],"INV":[{"chainName":"INV","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"UFC":[{"chainName":"UFC","canWithdraw":true,"fee":10.0,"mainChainName":"MAIN-UFC","canDeposit":true}],"QNT":[{"chainName":"QNT","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"XLM":[{"chainName":"XLM","canWithdraw":true,"fee":0.1,"mainChainName":"XLM","canDeposit":true}],"DDX":[{"chainName":"DDX","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"LINK":[{"chainName":"LINK","canWithdraw":true,"fee":0.5,"mainChainName":"ETH","canDeposit":true}],"DORA":[{"chainName":"DORA","canWithdraw":true,"fee":1.5,"mainChainName":"ETH","canDeposit":true}],"QTUM":[{"chainName":"QTUM","canWithdraw":true,"fee":0.01,"mainChainName":"QTUM","canDeposit":true}],"QFIL":[{"chainName":"QFIL","canWithdraw":true,"fee":1.0,"mainChainName":"ETH","canDeposit":true}],"UFO":[{"chainName":"UFO","canWithdraw":false,"fee":0.01,"mainChainName":"UFO","canDeposit":false}],"POLS":[{"chainName":"POLS","canWithdraw":false,"fee":20.0,"mainChainName":"ETH","canDeposit":true}],"KSM":[{"chainName":"KSM","canWithdraw":true,"fee":0.01,"mainChainName":"KSM","canDeposit":true}],"FIL":[{"chainName":"FIL","canWithdraw":true,"fee":0.002,"mainChainName":"FIL","canDeposit":true}],"POLY":[{"chainName":"POLY","canWithdraw":true,"fee":80.0,"mainChainName":"ETH","canDeposit":true}],"GALA":[{"chainName":"GALA","canWithdraw":true,"fee":120.0,"mainChainName":"ETH","canDeposit":true}],"BRC":[{"chainName":"BRC","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":false}],"BAR":[{"chainName":"BAR","canWithdraw":false,"fee":0.5,"mainChainName":"BAR","canDeposit":false}],"RAD":[{"chainName":"RAD","canWithdraw":true,"fee":2.0,"mainChainName":"ETH","canDeposit":true}],"BAT":[{"chainName":"BAT","canWithdraw":true,"fee":40.0,"mainChainName":"ETH","canDeposit":true}],"LEO":[{"chainName":"LEO","canWithdraw":true,"fee":5.0,"mainChainName":"ETH","canDeposit":true}],"RAI":[{"chainName":"RAI","canWithdraw":true,"fee":13.0,"mainChainName":"ETH","canDeposit":true}],"ARPA":[{"chainName":"ARPA","canWithdraw":true,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"XMR":[{"chainName":"XMR","canWithdraw":true,"fee":0.001,"mainChainName":"XMR","canDeposit":true}],"COTI":[{"chainName":"COTI","canWithdraw":false,"fee":100.0,"mainChainName":"ETH","canDeposit":false}],"ALCX":[{"chainName":"ALCX","canWithdraw":false,"fee":0.1,"mainChainName":"ETH","canDeposit":true}],"GUCS":[{"chainName":"GUCS","canWithdraw":false,"fee":0.01,"mainChainName":"GUCS","canDeposit":false}]}}
```

Reply:
On a sidenote – it would be great if we could implement `fetchFundingFees` for ZB or utilize that endpoint in `fetchCurrencies`.

Reply:
I have https://trade.zb.work/api set as the private api URL for spot:
'public': 'https://api.zb.work/data',
'private': 'https://trade.zb.work/api',
If I moved getFeeInfo to the public section wouldn't it be matched to https://api.zb.work/data ?
I believe this endpoint is already utilized in fetchCurrencies as well 
Let me know if there's something I'm missing, or if this part of the code can actually be left as is, or if I should reintroduce the trade section along with private and public

Reply:
Ah right, i missed that private base url. I think we can leave it as is for now. Thanks!